TelCo.ColorCoder/ColorCoder.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
```

Comment:
This namespace holds the colors, the printing and the conversion functions. Can there be a more descriptive name instead of `ColorCoder.cs`?

Reply:
I have updated the name of the namespace to `TelCo.WireColoringSystem`. Also I have made a more appropriate directory structure for the code.